test/common/grpc/grpc_client_integration_test_harness.h

```diff
@@ -28,7 +29,7 @@
 #include "test/integration/fake_upstream.h"
 #include "test/mocks/grpc/mocks.h"
 #include "test/mocks/local_info/mocks.h"
-#include "test/mocks/server/mocks.h"
+#include "test/mocks/server/transport_socket_factory_context.h"
```

Comment:
`Grpc::StatNames` is an incomplete type here (https://dev.azure.com/cncf/envoy/_build/results?buildId=44924&view=logs&j=ec9f720e-f904-5f65-4ba4-f371a013344f&t=ee47c280-92b1-5c80-3a10-d3d23a58a062&l=3606); we need to #include the header providing that name. server/mocks.h probably indirectly #included it before.

Reply:
yes, server/mocks.h previously #include "common/grpc/context_impl.h" which providing that name. i have sent a pr resolve this.